src/murmur/Meta.cpp

```diff
@@ -458,49 +467,45 @@ void MetaParams::read(QString fname) {
 	}
 #endif
 
-	if (! QSslSocket::supportsSsl()) {
-		qFatal("Qt without SSL Support");
-	}
-
 	{
 		QList<QSslCipher> ciphers = MumbleSSL::ciphersFromOpenSSLCipherString(qsCiphers);
 		if (ciphers.isEmpty()) {
 			qFatal("Invalid sslCiphers option. Either the cipher string is invalid or none of the ciphers are available: \"%s\"", qPrintable(qsCiphers));
 		}
 
+		QList<QSslCipher> filtered;
+#if !defined(USE_QSSLDIFFIEHELLMANPARAMETERS)
 		// If the version of Qt we're building against doesn't support
 		// QSslDiffieHellmanParameters, then we must filter out Diffie-
 		// Hellman cipher suites in order to guarantee that we do not
 		// use Qt's default Diffie-Hellman parameters.
-		QList<QSslCipher> filtered;
-#if !defined(USE_QSSLDIFFIEHELLMANPARAMETERS)
-		foreach (QSslCipher c, ciphers) {
-			if (c.keyExchangeMethod() == QLatin1String("DH")) {
-				continue;
+		{
+			QList<QSslCipher> filtered;
+			foreach (QSslCipher c, ciphers) {
+				if (c.keyExchangeMethod() == QLatin1String("DH")) {
+					continue;
+				}
+				filtered << c;
 			}
-			filtered << c;
-		}
-		if (ciphers.size() != filtered.size()) {
-			qWarning("Warning: all cipher suites in sslCiphers using Diffie-Hellman key exchange "
-			         "have been removed. Qt %s does not support custom Diffie-Hellman parameters.",
-				 qVersion());
+			if (ciphers.size() != filtered.size()) {
+				qWarning("Warning: all cipher suites in sslCiphers using Diffie-Hellman key exchange "
+				         "have been removed. Qt %s does not support custom Diffie-Hellman parameters.",
+					 qVersion());
+			}
+
+			qlCiphers = filtered;
```

Comment:
No, it is actually the correct one that is used. However, you are correct that there is a duplicate.
I moved the filtered var into the block when I refactored.